protos/sources/vega/vega.proto

```diff
@@ -478,6 +478,10 @@ enum OrderError {
   ORDER_ERROR_POST_ONLY_ORDER_WOULD_TRADE = 49;
   // Post order would trade
   ORDER_ERROR_REDUCE_ONLY_ORDER_WOULD_NOT_REDUCE_POSITION = 50;
+  // Isolated margin check failed
+  ORDER_ERROR_ISOLATED_MARGIN_CHECK_FAILED = 51;
```

Comment:
I think you might need to add these to the `OrderRejectionReason` enum in the graphQL schema.

Reply:
added